api/tests.yaml

```diff
@@ -0,0 +1,161 @@
+openapi: 3.0.0
+definitions:
+  Test:
+    type: object
+    properties:
+      id:
+        type: string
+        format: uuid
+        readOnly: true
+      name:
+        type: string
+      description:
+        type: string
+      serviceUnderTest:
+        type: object
+        properties:
+          request:
+            $ref: "./http.yaml#/definitions/HTTPRequest"
+      definition:
+        type: object
+        $ref: "#/definitions/TestDefinition"
+        description: Definition of assertions that are going to be made
+      referenceTestRun:
+        $ref: "#/definitions/TestRun"
+
+  TestDefinition:
+    type: object
+    properties:
+      definitions:
+        type: array
+        items:
+          type: object
+          properties:
+            selector:
+              type: string
+            assertions:
+              type: array
+              items:
+                $ref: "#/definitions/Assertion"
+    example:
+      definitions:
+        - selector: 'span[tracetest.span.type="http"]'
+          assertions:
+          - attr: tracetest.span.duration
+            comparator: "<"
+            expected: "2000"
+          - attr: http.status_code
+            comparator: "="
+            expected: "200"
+
+  Assertion:
+    type: object
+    properties:
+      id:
+        type: string
+        format: uuid
+        readOnly: true
+      attribute:
+        type: string
+      comparator:
+        type: string
+      expected:
+        type: string
+
+  TestRun:
+    type: object
+    properties:
+      id:
+        type: string
+        format: uuid
+        readOnly: true
+      testId:
+        type: string
+        format: uuid
+        readOnly: true
+      traceId:
+        type: string
+        readOnly: true
+      spanId:
```

Comment:
What is spanId in reference to a TestRun?

Reply:
both TraceID and SpanID are generated by us and sent in the triggering transaction